interface/utils.py

```diff
@@ -57,3 +47,30 @@ def random_code(length, vocabulary=vocabulary_64):
 def is_true(value):
     text = (value or '').lower().strip()
     return text in ['1', 'yes', 'true', 'on', 'enabled']
+
+    
+def get_url_base(link):
+    m = re.match(r'https://github.com/(?P<org>[^/]+)/(?P<repo>[^/]+)/?$',
+                    link.assignment.repo_url)
```

Comment:
The `link.assignment.repo_url` should start right below `r`. Something like:
```
m = re.match(r'https://github.com/(?P<org>[^/]+)/(?P<repo>[^/]+)/?$',
             link.assignment.repo_url)
```

I suggest using `flake8` as linter. We want the code to respect the `pep8` standard.